project1/application.py

```diff
@@ -45,6 +51,25 @@ def login():
         return "Please Login using your Credentials"
 
 
+@app.route("/search")
+def search():
+    return render_template('search.html')
+
+
+@app.route("/sresults", methods=["POST"])
+def sresults():
```

Comment:
Place both post and get methods in same route, based on request method implement respective block of code using if and else block.

Reply:
@krajeshkumar11 
Sir, the file is updated in the Search-Book Branch. Kindly check now.

Reply:
@Shereen789 good work can merge with master.